_profiles/roy_frank_cruz.md

```diff
@@ -0,0 +1,25 @@
+---
+# Required:
+title: "Roy Frank Cruz"     # first (middle) last
+country: US              # your country of residence (2 capital letters, e.g. US, GB, DE)
+
+# Optional (please add github or gravatar for your picture)
+github:     55238184                          # your github ID (not full url)
```

Comment:
Is this github id right? Same for gitlab id and linkedin id. Is this may the numerical id below instead of the handle?
![image](https://github.com/HSF/hsf.github.io/assets/4656391/2e6c60cb-452e-4b67-be1e-5b52ac7e8376)
In any case, the link to github.com/<number> won't work.

Reply:
Thanks! I've reached out to Roy to clarify

Reply:
@wdconinc Ready for review